src/js/actions/tools.js

```diff
@@ -484,14 +484,18 @@ define(function (require, exports) {
     var handleToolModalStateChanged = function (event) {
         var modalState = (event.state._value === "enter"),
             changeStatePromise = this.transfer(changeModalState, modalState),
+            policyStore = this.flux.stores.policy,
+            toolsIdsThatSuspendPolicies = ["txBx", "arwT", "Rect"],
```

Comment:
before @iwehrman has the chance to point it out; Set may fit better here than Array.